### ISSUE TITLE
Clicking without dragging doesn't update newIndex

### ISSUE THREAD

**Issue by fmapilis** [clauderic/react-sortable-hoc#11]
It seems like the SortableContainer's newIndex only resets when onSortMove is triggered, which doesn't seem to get fired if you just click a SortableElement without dragging it (which makes sense). Just clicking a SortableElement moves the element to the last newIndex that was set in that container (or 0 if it wasn't) and causes some funky sorting. 

The examples ([like this one](http://clauderic.github.io/react-sortable-hoc/#/basic-configuration/basic-usage)) show this happening too when you click an item besides Item 0.

**Comment by fmapilis** [comment]
My current workaround is to add `isDragging` to the state updates to stop onSortEnd from updating if the element hasn't actually been dragged yet:

```javascript
  onSortMove = (e) => {
    this.setState({isDragging: true})
  }
  onSortEnd = ({oldIndex, newIndex}) => {
    if(!this.state.isDragging)
      return

    this.setState({
        items: arrayMove(this.state.items, oldIndex, newIndex),
        isDragging: false
    })
  }
```

**Comment by clauderic** [comment]
Hmm yeah that's pretty silly 😂

**Comment by clauderic** [comment]
Fixed in `0.0.2` release
